### ISSUE TITLE
Cloud storage does not work between Windows and Linux

### ISSUE THREAD

**Issue by irgendsontyp** [gorhill/uMatrix#939]
I am using uMatrix on Arch Linux, all software is up to date. I enabled cloud storage support, connected Firefox with my sync account, and selected "Add ons" in the account preferences. I uploaded my current config to the cloud and synced my Firefox account after that. Then I reboot to Windows 10, current Firefox. Same settings here, sync account is connected. After having synchronized my account, the uMatrix dashboard does not show the information that I have uploaded a config from Linux. Thus, I cannot import the rules to Windows 10 that I previously exported via Arch Linux. But when I open a virtual machine running Ubuntu with the latest version of Firefox, same settings, uMatrix' dashboard displays the uploaded config and I can download it. Why does it work from Arch Linux to Ubuntu but not from Arch Linux to Windows 10?

Please let me know if you require more detailed information. I would really love to be able to sync between Linux and Windows as, unfortunately, I sometimes "have" to use Windows and it would be nice to have all my rules from Arch Linux there, which I mainly use.

**Comment by gorhill** [comment]
You will have to investigate this on your side, you are better placed than me. uBO just use the `storage.sync` API, all the rest is handled by Firefox.

**Comment by koneu** [comment]
Works fine for me between FF58 on Arch, Nightly on Leap and FF58 on Win8.

**Comment by irgendsontyp** [comment]
I figured out that it works for me if I set the value of the about:config entry "services.sync.log.appender.file.logOnSuccess" to "true" on Windows as well as on Linux (was false initially on both of those).

I found this out just by playing around with the about:config entries which I found filtering for "sync.".

I really do not know why it works because this does not make any sense to me. Maybe there is some kind of routine that clears a cache if you set this preference? Would be strange though.

Maybe someone knows why this works?
